oauthenticator/hydroshare.py

```diff
@@ -161,6 +162,12 @@ def authenticate(self, handler, data=None):
         # get the username variable from the response
         username = resp_json["username"]
         
+        # save toekn to users home dir
+        fname = '/userspace/%s/.hs_auth' % (username)
```

Comment:
the userspace is configurable...
https://github.com/hydroshare/hydroshare-jupyterhub/blob/develop/deploy/config/jupyterhub_config.py#L53

And since you're in here, toekn in the comment above.

Reply:
Looks like JUPYTER_USERSPACE_DIR was the right one.